src/dynamic_boundary_conditions/tide_data.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 16:30:21 2022.
+
+@author: alo78
+"""
+
+import os
+from dotenv import load_dotenv
+from datetime import date, datetime, timedelta
+import pandas as pd
+import requests
+
+
+
+def get_Tide_data(NIWA_API_KEY, lat, long, total_days, startDate, datum):
+    """ returns dataframe that has been concatenated with new data as max output of days data from API is 31"""
+    modulo = total_days % 30
+    if modulo == 0:
+        numberOfDays = 30
+        months = (total_days // 30) - 1
+    else:
+        numberOfDays = modulo
+        months = total_days // 30
+
+    parameters = {'lat': lat, 'long': long, 'apikey': NIWA_API_KEY, 'numberOfDays': numberOfDays,
+                  'startDate': startDate, 'datum': datum}
+
+    # A GET request for NIWA API
+    NIWA_API = requests.get('https://api.niwa.co.nz/tides/data', params=parameters)
+
+    if NIWA_API.status_code != 200:
+        print("API ERROR: ", NIWA_API.status_code)
+        print("Please check input and try again!")
+        quit()
+    else:
+        Tide_df = pd.DataFrame(NIWA_API.json()['values'][2:])
```

Comment:
```suggestion
        Tide_df = pd.DataFrame(NIWA_API.json()['values'])
```

Keep the first two rows as these are data for the current date as well.

Reply:
Keep the first two rows as these are data for the current date as well.